index.js

```diff
@@ -0,0 +1,53 @@
+const list = ['掃除', '買い物', '散歩'];
+const display_text = `現在持っているタスク一覧\n`;
+function list_display() {
+  console.log(display_text);
+  for (let i = 0; i < list.length; i++) {
+    console.log(`${i}:${list[i]}`);
+  }
+};
+
+function list_add(task) {

```

Comment:
こちらも `camelCase` の方がいいですね。

```suggestion
function listAdd(task) {

```